paper-ieee-trans.tex

```diff
@@ -201,10 +201,10 @@ \subsection{DataGrid}
 communities (50 use cases from the three scientific area), and 3) the lack of a body
 of knowledge for academic software engineering.
 
-The Agile manifesto \cite{agile-manifesto} was by then an emerging methodology, thus not
+The Agile manifesto \cite{agile-manifesto} was by then an emerging methodology, not yet
 considered by the project, and as such the project's
-development design suffered from the methods, tools, techniques and best
-practices emerging from the new discipline of software engineering
+development design suffered from the lack of the methods, tools, techniques and best
+practices emerging from these new discipline of software engineering
```

Comment:
these -> this

Reply:
change to this
agree with the other changes

Reply:
ok

Reply:
ok

Reply:
ok